packages/lit-html/src/test/is-server_test.ts

```diff
@@ -0,0 +1,14 @@
+/**
+ * @license
+ * Copyright 2022 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import {isServer} from 'lit-html/is-server.js';
+import {assert} from '@esm-bundle/chai';
+
+suite('is-server', () => {
+  test('isServer is falsey', () => {
+    assert.notOk(isServer);
```

Comment:
Go ahead and do a stricter check. We want this to be actually `false`

Reply:
Done!